### DISCUSSION TITLE
Deutsche Übersetzung

### DISCUSSION THREAD

**Discussion by AlexJacu** [skipperro/enpal-homeassistant#10] [Ideas]
Bitte Deutsche Übersetzung hinzufügen.
[de.json](https://github.com/user-attachments/files/18101820/de.json)

Bitte auch enpal-homeassistant/custom_components/enpal
/sensor.py
übersetzen, bzw. aus translation Dateien benamen.

**Comment by skipperro** [comment]
Gute Idee! Bitte um Pull Request.
